GOVERNANCE.md

```diff
@@ -144,6 +144,44 @@ Contributions can be:
 * Participation in other projects, teams, and working groups of the Node.js
   organization.
 
+Collaborators should be people volunteering to do unglamorous work because it's
+the right thing to do, they find the work itself satisfying, and they care about
+Node.js and its users. People should get collaborator status because they're
```

Comment:
@jasnell could you emphasize then that a collaborator is not just someone working on nodejs/node

Reply:
We list what contributions are expected to be relevant: https://github.com/nodejs/node/blob/df883b0c7234cb44bef3175cd3e6a997c830db05/GOVERNANCE.md#L173-L190

Does that address your concern?

Reply:
I don't think so, this is what we had already. Maybe just using **bold** around those lines is enough. But it is fine if y'all believe leave as it is.

Reply:
I lean towards keeping this as is but can be convinced. I think such changes could come in a second PR tho.

Reply:
Fine by me!